templates/common/gcp/files/usr-local-bin-update-dns-server.yaml

```
@@ -0,0 +1,23 @@
+mode: 0755
+path: "/usr/local/bin/update-dns-server"
+contents:
+  inline: |
+    #!/bin/bash
+    
+    echo "Executing update-dns-server"
+
+    mkdir -p /etc/NetworkManager/conf.d
+
+    echo "#Added by OpenShift" > /etc/NetworkManager/conf.d/dns-servers.conf
+    echo "[global-dns-domain-*]" >> /etc/NetworkManager/conf.d/dns-servers.conf
+
+    # 169.254.169.254 is the IP address of the GCP metadata server
+    echo "servers=$(ip --json route get 8.8.8.8 | jq /".[0].prefsrc/"" >> /etc/NetworkManager/conf.d/dns-servers.conf
```

I think part of this line was accidentally dropped in https://github.com/openshift/machine-config-operator/compare/4269c5227c055473d524166b5f2b999770f06363..a44a4abb1e6ed9904b85e94f26e874de121c12f6

```suggestion
    echo "servers=$(ip --json route get 8.8.8.8 | jq ".[0].prefsrc"),169.254.169.254" >> /etc/NetworkManager/conf.d/dns-servers.conf
```

It's not clear to me what the object of this command is though... we're trying to find the IP address of an interface that's connected to the outside world? Could we not use localhost?
And the 169.254.169.254 is replacing the one we presumably get from DHCP so it doesn't get overwritten? (I wonder if setting this for just the cluster domain instead of `global-dns-domain-*` might allow us to drop this and use the same config file across different platforms.)

---

Maybe https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/openshift_node/files/networkmanager/99-origin-dns.sh#L5-L9 is the explanation for why we need this.